degirum_tools/zone_count.py

```diff
@@ -55,14 +68,24 @@ def __init__(
         self,
         polygon: np.ndarray,
         frame_resolution_wh: Tuple[int, int],
-        triggering_position: AnchorPoint,
+        triggering_positions: Optional[List[AnchorPoint]],
+        bounding_box_scale: float = 1.0,
+        iopa_threshold: Optional[float] = 0.0,

```

Comment:
Please be advised that `Optional` annotation means that this parameter is allowed to be `None`. If it is **not** allowed to be `None`, then remove `Optional`.
Here this parameter is not optional, but has default value. It is different thing.